### DISCUSSION TITLE
2005/05/05/%E5%88%AB%E4%BA%BA%E7%9A%84%E8%AE%A1%E6%95%B0%E5%99%A8%EF%BC%8C%E5%85%88%E6%94%BE%E4%B8%80%E4%B8%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#20] [Announcements]
# 2005/05/05/%E5%88%AB%E4%BA%BA%E7%9A%84%E8%AE%A1%E6%95%B0%E5%99%A8%EF%BC%8C%E5%85%88%E6%94%BE%E4%B8%80%E4%B8%8B

https://blog.qizhen.xyz/2005/05/05/%E5%88%AB%E4%BA%BA%E7%9A%84%E8%AE%A1%E6%95%B0%E5%99%A8%EF%BC%8C%E5%85%88%E6%94%BE%E4%B8%80%E4%B8%8B

**Comment by ruanqizhen** [comment]
Acuto
2005年05月8日 11:08 编辑
专业

回复
Unknown
2005年05月6日 07:29 编辑
呵～笑什麼？你也長鬢毛嗎？

回复
灵旗
2005年05月6日 02:35 编辑
你好，谢谢你在我的bolg上面留言，我也循着你的足迹来到了这里！我发现你是属于技术性人才！我很想跟你学习一下blog留言技巧，包括添加计数器、留言中使用的窗体等！希望你能赐教！

回复
Unknown
2005年05月5日 21:27 编辑
这个计数器我不能按ctrl+c,点了它又出现其他网站了…怎么回事?我按右键"图片另存为"不行吗

回复
Unknown
2005年05月5日 16:24 编辑
emm…做好了,lol不知它会不会动…~~谢谢你

回复
